script.js

```diff
@@ -0,0 +1,40 @@
+const prompt = require('prompt-sync')();
+
+let input = prompt ('Please input a score between 0 and 100: ');
+
+function checkInput(){
+    if (input >= 0 && input <= 30 || input >= 31 && input <= 60 || input >= 61 && input <= 100  && input !='') {
+       performOperation();
+    } else {
+        console.log('Please insert a valid score!');
+    }
+
+    resetCheckInput();
+  }
+
+checkInput();
+
+
+function performOperation(){
+    
+    if (input >=0 && input <=30) {
+        console.log("Try Harder your score is low!");
+        return input;
```

Comment:
Check your return I dont think it should be input but rather you can set what your result (what you are console logging) as a variable and return it, but that return is not necessary, check same for all other returns

Reply:
Done